packages/services/src/assets/sip9-asset.utils.ts

```diff
@@ -59,6 +59,29 @@ export function getNonFungibleTokenId(hex: string): number {
   return clarityValue.type === 'uint' ? Number(clarityValue.value) : 0;
 }
 
+const leatherIpfsUrl = 'https://leather.quicknode-ipfs.com/ipfs/';
+
+function getLeatherIpfsUrl(url: string): string {
```

Comment:
Mostly nitpicks:
* something like `toLeatherIpfsUrl` or `rewriteToLeatherIpfsUrl` would be clearer, since this is technically a transformation.
* Comments are redundant, the code is pretty self-explanatory
* early return: `if (!url.includes('/ipfs/')) return url;`